styles/products.css

```diff
@@ -0,0 +1,30 @@
+.products-container {
+    display:flex;
+}
+
+.products-filters {
+    width: 15rem;
+}
+.filter{
+    border-bottom: 1px solid var(--primary);
+    margin-bottom: 1rem;
+    padding: 0.5rem;
+}
+
+.filter-brand{
+    display:block;
+}
+
+.products-listings{
+    padding: -1rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
its right , you can also try repeat(3,fr) in grid-template-columns (tell me if i wrong )

Reply:
repeat(3, 1fr) ;

Reply:
Yes, you are right 👍🏽 repeat(3, 1fr)